index.bs

```diff
@@ -6064,6 +6080,287 @@ However, [=authenticators=] that do not utilize [[!FIDO-CTAP]] do not necessaril
 :: [=largeblob|This extension=] directs the user-agent to cause the large blob to be stored on, or retrieved from, the authenticator. It thus does not specify any direct authenticator interaction for [=[RPS]=].
 
 
+## Device-bound public key extension (<dfn>devicePubKey</dfn>) ## {#sctn-device-publickey-extension}
+
+This [=authenticator extension|authenticator=] [=registration extension=] and [=authentication extension=] provides a [=[RP]=] with a "device continuity" signal. This is done by creating a [=[RP]=]-specific [=hardware-bound device key pair=] on the [=client device=], if such a key pair does not already exist for the [=user credential=] being created or exercised, and returning the attested [=device public key=] along with a signature by the [=device private key=] to the [=[RP]=]. This is done each time this [=devicePubKey=] extension is included with either a {{CredentialsContainer/create()|navigator.credentials.create()}} or {{CredentialsContainer/get()|navigator.credentials.get()}} call.
```

Comment:
Should this not be "Credential-Specific" instead of "RP-specific" ?

If this is only RP-specific and not credential-specific, this could be used to correlate different credentials within the same RP.

Reply:
Good catch, I agree.

Reply:
Done in [ccfd0b4](https://github.com/w3c/webauthn/pull/1663/commits/ccfd0b4dfe95245d4771c290ed71c225142fd98d)